README.md

```diff
@@ -1,11 +1,10 @@
-# Kafka Airflow Provider
-
-
-> **Warning**
-
-> This package has been deprecated after being accepted to OSS Airflow. Please use apache-airflow[apache.kafka] instead if you're looking for a supported kafka provider. 
+## ⚠️ Discontinuation of project
+> This project is no longer actively maintained by Astronomer, but we’d love to see it live on in the community. While Astronomer has paused development and is not accepting new contributions, bug fixes or releases, the code is still here for you to explore, fork and adapt under the terms of its license. 
```

Comment:
PS: need to check if this is rendering well
```suggestion
> Astronomer [donated this provider](https://github.com/apache/airflow/pull/30175) to the official Apache Airflow repository in March 2023. Since then, the original Astronomer repository and its PyPI package are no longer actively maintained. For more information on the new Kafka package:
> * PyPI package: [apache-airflow-providers-apache-kafka](https://pypi.org/project/apache-airflow-providers-apache-kafka/) 
> * Documentation: [Apache Airflow Kafka official documentation](https://airflow.apache.org/docs/apache-airflow-providers-apache-kafka/stable/index.html)
> * Source code: [Apache Airflow Kafka provider source code](https://github.com/apache/airflow/tree/main/providers/apache/kafka)
```